release/models/system/openconfig-aaa.yang

```diff
@@ -32,7 +32,14 @@ module openconfig-aaa {
     Portions of this model reuse data definitions or structure from
     RFC 7317 - A YANG Data Model for System Management";
 
-  oc-ext:openconfig-version "0.4.1";
+  oc-ext:openconfig-version "0.4.2";
```

Comment:
Believe any adjustments to version/revision will be done at time of release to batch fixes/changes so unlikely unnecessary here

Reply:
Thanks. I have removed revision changes.